src/test/FuzzerImpl.cpp

```diff
@@ -190,90 +190,20 @@ generateStoredLedgerKeys(StoredLedgerKeys& keys)
     static_assert(NUM_UNVALIDATED_LEDGER_KEYS < NUM_STORED_LEDGER_KEYS,
```

Comment:
I think you're making this more confusing than it needs to be. I stared at lines 1166,1168 for too long trying to understand what things would get overwritten. Use the same technique we've used in other places: make this accept iterators (which is idiomatic for functions like this). In the `TransactionFuzzer` you can generate the stored keys after you have added the ones created by set-up so no overwriting is required. In the `OverlayFuzzer` you can just generate the entire vector.

I'm pretty sure this would make everything quite a bit clearer. But let me know if you disagree.

Reply:
Alright, I've taken a crack at an iterator interface, which allows us to avoid generating any pseudorandom entries that will just be overwritten by the ones from setup.